src/com/trollworks/gcs/skill/SkillOutline.java

```diff
@@ -157,6 +158,71 @@ public void increment() {
 	}
 
 	@Override
+	public boolean canIncrementSkillLevel() {
+		return canIncrement();
+	}
+
+	@Override
+	public boolean canDecrementSkillLevel() {
+		return canDecrement();
+	}
+
+	@SuppressWarnings("unused")
+	@Override
+	public void incrementSkillLevel() {
+		List<RowUndo> undos = new ArrayList<>();
+		for (Skill skill : new FilteredIterator<>(getModel().getSelectionAsList(), Skill.class)) {
+			if (!skill.canHaveChildren()) {
+				int basePoints = skill.getPoints() + 1;
+				int maxPoints = basePoints + (skill.getDifficulty() == SkillDifficulty.W ? 12 : 4);
+				int oldLevel = skill.getLevel();
+				RowUndo undo = new RowUndo(skill);
+				for (int points = basePoints; points < maxPoints; ++points) {
+					skill.setPoints(points);
+					if (skill.getLevel() > oldLevel) {
+						break;
+					}
+				}
+				if (undo.finish()) {
+					undos.add(undo);
+				}
+			}
+		}
+		if (!undos.isEmpty()) {
+			repaintSelection();
+			new MultipleRowUndo(undos);
+		}
+	}
+
+	@SuppressWarnings("unused")
+	@Override
+	public void decrementSkillLevel() {
+		List<RowUndo> undos = new ArrayList<>();
+		for (Skill skill : new FilteredIterator<>(getModel().getSelectionAsList(), Skill.class)) {
+			if (!skill.canHaveChildren()) {
+				int basePoints = Math.max(0, skill.getPoints() - 1);
+				int minPoints = Math.max(0, basePoints - (skill.getDifficulty() == SkillDifficulty.W ? 12 : 4));
```

Comment:
The same comment from above seems to apply here, but in reverse. There is the edge case of not being able to go below the point of no points spent, but that's different than what you're doing here.